packages/remote-feature-flag-controller/src/client-config-api-service/abstract-client-config-api-service.ts

```diff
@@ -1,9 +1,20 @@
-import type { PublicInterface } from '@metamask/utils';
+import type { ServicePolicy } from '@metamask/controller-utils';
 
-import type { ClientConfigApiService } from './client-config-api-service';
+import type { ServiceResponse } from '../remote-feature-flag-controller-types';
 
 /**
  * A service object responsible for fetching feature flags.
  */
-export type AbstractClientConfigApiService =
-  PublicInterface<ClientConfigApiService>;
+export type IAbstractClientConfigApiService = Partial<
```

Comment:
I opted not to use `PublicInterface` here because it aligns with `AbstractTokenPricesService`, and it's more explicit IMO.